Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "interpoli"
+version = "0.1.0"
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+description = ""
+keywords = ["graphics", "animation"]
+categories = ["graphics"]
+repository = "https://github.com/linebender/interpoli"
+readme = "README.md"
+# Keep in sync with RUST_MIN_VER in .github/workflows/ci.yml, with the relevant README.md files
+# and with the MSRV in the `Unreleased` section of CHANGELOG.md.
+rust-version = "1.75"
+
+[lints]
+clippy.doc_markdown = "warn"
+clippy.semicolon_if_nothing_returned = "warn"
+
+[features]
+
+[package.metadata.docs.rs]
+all-features = true
+
```

Comment:
We should specify a single target here. E.g. https://github.com/linebender/xilem/blob/c6c1b716405fa2c024e0d69eff22567d017f38aa/xilem_core/Cargo.toml#L25-L29

Reply:
I don't think that's necessary, but I've added it.

Reply:
The important part of the line is more:
```suggestion
# Interpoli is entirely platform-agnostic, so only display docs for one platform
targets = []

```

But it's also fine not to have.